src/app/app.module.ts

```diff
@@ -11,8 +11,9 @@ import { AdminService } from './services/admin.service';
 import { StudentService } from './services/student.service';
 import { LoggedComponent } from './logged/logged.component';
 import { StudentComponent } from './logged/student/student.component';
-import { AddComponent } from './logged/student/add/add.component';
+import { FormComponent } from './logged/student/form/form.component';
 import { AllComponent } from './logged/student/all/all.component';
+import { AddComponent } from './logged/student/add/add.component';
```

Comment:
Name of the import not explicit, could be redundant. StudentAddComponent could be a better choice.